.all-contributorsrc

```diff
@@ -709,6 +709,24 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "solpark",
+      "name": "Sol Park",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/30510616?v=4",
+      "profile": "https://github.com/solpark",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "michaelromani",
```

Comment:
@solpark Can you confirm that adding "Michael Romani" is intentional?

Reply:
Yes, worked on this together!